tests/test_signing.py

```diff
@@ -1,12 +1,24 @@
+import json
 from datetime import datetime, timedelta, timezone
 
 import itsdangerous
 import pytest
 from pydantic import ValidationError
+from pydantic.json import pydantic_encoder
 
 from hatch import signing
 
 
+def create_raw_token(value, signer=None):
+    """To be used to create bad tokens that AuthToken won't let you
+    create."""
```

Comment:
I don't like the run onto a second line here… is that black wrapping it? (I'll suck it up if it is…)

Reply:
no, vim wrapping, fixed.